src/status_im/subs/general.cljs

```diff
@@ -292,3 +292,11 @@
  :<- [:toasts]
  (fn [toasts [_ toast-id & cursor]]
    (get-in toasts (into [:toasts toast-id] cursor))))
+
+(re-frame/reg-sub
+ :alert-banners-top-margin
```

Comment:
can we have an accompanying test for this?

Reply:
do we have no qualificatio for these subs? e.g `:general/alert-banners-top-margin`